src/modules/LanItems.rb

```diff
@@ -430,7 +430,13 @@ def ReplaceItemUdev(replace_key, new_key, new_val)
       operator = new_key == "NAME" ? "=" : "=="
       current_rule = getUdevFallback
```

Comment:
I'm fine with that POV so just check that we have a test for this case and we don't let some corner case uncovered, I can check if if you want.